### PR TITLE
Added the whl file for 1.0.0 version of aksarc cli (renamed from akshybrid)

### PR DESCRIPTION
**Pull request by HariPrasadV**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/index.json [hunk 1]
```diff
@@ -9697,6 +9697,63 @@
                 "sha256Digest": "a80a939563355331e53c54297710ec3198de460d04d357351a6decee5e6a64d1"
             }
         ],
+        "aksarc": [
+            {
+                "downloadUrl": "https://hybridaksstorage.z13.web.core.windows.net/HybridAKS/CLI/aksarc-1.0.0-py3-none-any.whl",
+                "filename": "aksarc-1.0.0-py3-none-any.whl",
+                "metadata": {
+                    "azext.isExperimental": true,
```
Comment (kairu-ms) [inline src/index.json:9705]: This tag is not used any more. For review version, please use version 1.0.0b1

Comment (HariPrasadV) [reply src/index.json:9705]: We are removing the isExperimental and isPreview flags, I'll update this PR soon

**Comment by HariPrasadV** [comment]
> Will you remove `akshybrid` extension?

akshybrid extension is already being used for our public preview, so we will continue to support it for a few more months until GA in January (after that we will have only aksarc going forward)

File: src/index.json (hunk 1 above)
Comment (HariPrasadV) [reply src/index.json:9705]: Updated it now :)

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**